velox/vector/ComplexVector.h

```diff
@@ -184,6 +184,16 @@ class RowVector : public BaseVector {
         nullCount_);
   }
 
+  void transferOrCopyTo(velox::memory::MemoryPool* pool) override {
+    BaseVector::transferOrCopyTo(pool);
+    for (auto& child : children_) {
+      child->transferOrCopyTo(pool);
+    }
+    if (rawVectorForBatchReader_) {
```

Comment:
We have rawVectorForBatchReader_ defined here?

Reply:
@xiaoxmeng Yes, it's a pre-existing data member of RowVector.